cron-jobs/checkForLastLogin.js

```diff
@@ -4,26 +4,32 @@ const { sendEmail } = require('~/utils/emailService')
 const emailSubject = require('~/consts/emailSubject')
 const { oneDayInMs } = require('~/consts/auth')
 
+const DAYS_TO_SEND_EMAILS = 173
+const DAYS_TO_DELETE_USER = 180
+
 const checkUsersForLastLogin = new CronJob('00 00 03 * * *', () => checkLastLogin())
```

Comment:
if in `removeUnverifiedUsers.js` you saved cronTime to a variable, then why didn't you do the same here?

Reply:
i did not know how to name this variable , i need to know  frequency of this string '00 00 03 * * *' - I was trying to find out it but did not managed this str seems to be to long ( maybe you can help me and say frequency of this cron? Thx in advance!

Reply:
The frequency is every day at 3 AM

Reply:
So you could call it `EVERY_THIRD_HOUR` or something similar to that.